test/e2e/support/reports.js

```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
+// const fs = require('fs');
+// const mkdirp = require('mkdirp');
 
-const xmlReports = `${process.cwd()}/reports/xml`;
+// const xmlReports = `${process.cwd()}/reports/xml`;
 
-// var createXmlDir = (function() {
-//     if (!fs.existsSync(xmlReports)) {
-//         mkdirp.sync(xmlReports);
-//     }
-// })();
+// // var createXmlDir = (function() {
+// //     if (!fs.existsSync(xmlReports)) {
+// //         mkdirp.sync(xmlReports);
+// //     }
+// // })();
 
-reporter.config({ targetDir: xmlReports });
+// reporter.config({ targetDir: xmlReports });
 
-module.exports = reporter;
+// module.exports = reporter;
```

Comment:
And this one, too.

Reply:
i have removed the contents of the entire file